packages/nimble-components/src/select/index.ts

```diff
@@ -293,16 +278,20 @@ export class Select extends FormAssociatedSelect implements ErrorPattern {
         next: Element[]
     ): void {
         const value = this.value;
-        this._options.forEach(o => {
+        this.options.forEach(o => {
             const notifier = Observable.getNotifier(o);
             notifier.unsubscribe(this, 'value');
+            notifier.unsubscribe(this, 'hidden');
+            notifier.unsubscribe(this, 'disabled');
         });
 
         super.slottedOptionsChanged(prev, next);
 
-        this._options.forEach(o => {
+        this.options.forEach(o => {
```

Comment:
While playing around in dev tools in storybook, I noticed that changing the content of the selected list option (or the placeholder when nothing is selected) does NOT reflect in the select. Can it / should it?

Since this is pre-existing behavior I don't think it needs to be changed as part of this PR but if we think it matters we could fix in a follow up.

Reply:
Going by how the native `select` works, a change to the content of a selected option _should_ update the `Select` display. I'll look into this for a follow up PR.

Reply:
@jattasNI, I'll note that if you update the content of an option by assigning its `textContent` a new value in the console, the `Select` display _does_ update. I imagine the only way to get it to work in a more complete way is to use a `MutationObserver`, which FAST does not do, and I'm unsure there's enough value there to implement that.

Reply:
OK, cool. I was updating it by editing the DOM in dev tools, which is not nearly as important a use case. I'm fine to leave the behavior as-is since the programmatic API works.

Though we should probably have a test to ensure the `textContent` case continues to work.

Reply:
It does seem kinda racy. Should maybe be a usage note in the select docs that the content of the placeholder option needs to be set before setting the attributes for the display text to be correct.

We could consider in the future just setting the MutationObserver up on the placeholder option instead of all options.

Reply:
I'm not sure I follow your suggestion for the usage docs. Do you simply mean that we should stress that when setting up a placeholder option that they should follow the convention of putting the contents in the markup, and not assigning it later? I'm not sure what the race condition is, and this isn't a placeholder issue, but a general one.

Reply:
I see, it's an issue for any selected option, placeholder or otherwise.

> I'm not sure what the race condition is

> they should follow the convention of putting the contents in the markup, and not assigning it later

Maybe race is the wrong word. But like the subject of this thread the order of operations is significant. Your display text won't be recognized in some cases. That should be documented.

Reply:
I've added the test @jattasNI suggested. I'll update docs as needed in a follow-up PR if there is, in fact, some agreed upon doc changes.